apps/mobile/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,27 @@
+platform :ios do
+  desc "Build and distribute iOS app"
+  lane :distribute_ios do
+    build_app(
+      scheme: "YourAppScheme" # Specify your app's scheme
```

Comment:
I'm not sure what to put for `scheme` ?

Reply:
<img width="465" alt="Screenshot 2024-12-20 at 7 56 01 AM" src="https://github.com/user-attachments/assets/c8e1bcc6-3377-4beb-bfff-3340eb1d764b" />

it should be this:

```
<string>myapp</string>
<string>io.leather.mobilewallet</string>
```

we should change `myapp` to `leather` and you can specify the full scheme as `leather://`. That will allow links to `leather://` to resolve to the application on iOS when followed by a user.